modules/aurora-mysql-resources/variables.tf

```
@@ -11,7 +11,7 @@ variable "aurora_mysql_component_name" {
 variable "ssm_passwords_enabled" {
```

```suggestion
variable "read_passwords_from_ssm" {
```
Like I said above, we have several other instances of variables with this or very similar names, and they all enable storing passwords in SSM. Let's use a more specifically descriptive name to indicate the different usage.

---

updated variable name and occurances